array-methods.js

```diff
@@ -1,14 +1,25 @@
 var dataset = require('./dataset.json');
-
+// console.log(dataset.bankBalances)
 /*
   create an array with accounts from bankBalances that are
   greater than 100000
   assign the resulting new array to `hundredThousandairs`
 */
-var hundredThousandairs = null;
+
+
+var hundredThousandairs = dataset.bankBalances.filter(function (element) {
+  return element.amount > 100000
+})
+// console.log(hundredThousandairs)
 
 // set sumOfBankBalances to be the sum of all value held at `amount` for each bank object
-var sumOfBankBalances = null;
+
+const sumOfBankBalances = dataset.bankBalances.map(function (element) {
+  return Number(element.amount)
+}).reduce(function (prev, current) {
```

Comment:
The normal convention is to put the `.reduce` on the next line but indented in two spaces. This is for all chained methods except for the first one which can stay how it is right now for the `.map`. Please make sure the rest of the file follows this convention.